src/test/resources/features/admin/create_teacher.feature

```diff
@@ -50,6 +51,7 @@ Feature: Admin page/ Teacher section functionality
     And Fill in all required fields
     And Click on 'Generate password' button
     And Save value from password input field
+    And Save value from email input field
     And Click on 'Save' button
     Then Check the teacher password is hashed in the DB
 
```

Comment:
Please remove unnecessary empty lines.

Reply:
done, and I left one empty row at the end, because git likes it :)